### PR TITLE
Kpt command order update

### PR DESCRIPTION
**Pull request by cartyc**
Updated the documentation around `kpt live init` and commented out the `config-control` namespace object due to it conflicting with the pre-provisioned namespace of the same name.

If the `config-control` namespace is not commented out `kpt live apply` can fail and require the end user to run `kpt live apply --inventory-police adopt` to get a successful deployment or a failure when using GitOps on the `alpha` version of Config Controller due to policy restrictions on modifying that Namespace.

### CODE REVIEW

**Comment by cartyc** [comment]
Will squash this on merge.

File: solutions/landing-zone/README.md
Comment (fmichaelobrien) [inline solutions/landing-zone/README.md:191]: LGTM
I'll need to test this locally

**Review by fmichaelobrien — APPROVED** [review]

File: solutions/landing-zone/README.md
Comment (cartyc) [reply solutions/landing-zone/README.md:191]: Will hold off on merging until you give the all clear.

Comment (fmichaelobrien) [reply solutions/landing-zone/README.md:191]: I might not get to this testing today, reworking a couple of my kcc lz orgs.  The change is good and critical - I can side test after it is merged.

Good to merge

**Pull request merged**
